library/src/main/java/com/heinrichreimersoftware/materialintro/app/SlideFragment.java

```diff
@@ -58,20 +58,30 @@ public void removeOnNavigationBlockedListener(OnNavigationBlockedListener listen
     }
 
     @Override
-    public boolean goToSlide(int position) {
-        return getIntroActivity().goToSlide(position);
+    public boolean goToSlide(int position, boolean forceScroll) {
+        return getIntroActivity().goToSlide(position, forceScroll);
     }
 
     @Override
     public boolean nextSlide() {
         return getIntroActivity().nextSlide();
     }
 
+    @Override
+    public boolean forceNextSlide() {
```

Comment:
I would remove the duplicated code in favor of a `nextSlide(boolean forceScroll)` method.
The old `nextSlide()` method should then call `nextSlide(false);`